packages/ui/src/composites/wui-card-select/styles.ts

```diff
@@ -0,0 +1,50 @@
+import { css } from 'lit'
+
+export default css`
+  button {
```

Comment:
I've noticed that we have quiet a few repeated styles on components that use buttons atm, would it make sense to put them under global styles on a `button`? If not all buttons need this, we could make a global class like `wui-button-base` or something and apply it where needed. 

Similarly for button focus / hover states etc.